<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>813</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Estimar</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: white;
	border-image: url(:/imagenes/assets/Image.png) 0 0 0 0 stretch stretch;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>681</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_titulo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Estimar Plan de Estudio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_ok">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QPushButton {
	width: 150px;
height: 60px;
left: 735px;
top: 324px;
color: white;
background: #FF7A00;
border-radius: 15px;
}

.QPushButton:hover {
	background: #083C87;
}

.QPushButton:pressed {
	background: #E6E6E6;
}</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_cancel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QPushButton {
	width: 150px;
	height: 60px;
	left: 735px;
	top: 324px;
	color: white;
	background: #808285;
	border-radius: 15px;
}

.QPushButton:hover {
	background: #083C87;
}

.QPushButton:pressed {
	background: #E6E6E6;
}</string>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_periodo">
       <property name="text">
        <string>Período a Estimar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radioButton_1">
       <property name="text">
        <string>Primer Semestre</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>Segundo Semestre</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_tipo_estimacion">
       <property name="text">
        <string>Tipo de Estimación:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="radioButton_4">
       <property name="text">
        <string>Estimación Final</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radioButton_3">
       <property name="text">
        <string>Estimación A Priori</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_error_ano">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>127</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #B00020;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_error_periodo">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>185</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #B00020;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_error_tipo_estimacion">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>245</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #B00020;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_error_plan">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>77</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #B00020;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item row="4" column="0">
      <widget class="QLabel" name="label_ano">
       <property name="text">
        <string>Año a Estimar:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QRadioButton" name="radioButton_7">
       <property name="text">
        <string>Civil 2019</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="input_ano">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="radioButton_6">
       <property name="text">
        <string>Civil 2014</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_archivo">
       <property name="text">
        <string>Plan:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radioButton_5">
       <property name="text">
        <string>Ejecución 2014</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
